etl/steps/data/garden/faostat/2025-03-17/additional_variables.py

```diff
@@ -1219,6 +1219,65 @@ def generate_net_exports_as_share_of_supply(tb_fbsc: Table) -> Table:
     return tb
 
 
+def generate_milk_per_cow(tb_qcl: Table) -> Table:
+    # FAOSTAT QCL used to have yield for milk (milk per cow), but it was removed at some point (that combination of item and element is empty in the 2025 release).
+    # Numerically, it seems that their old milk yield indicator coincides with production / slaughtered animals.
```

Comment:
Do you mean to write slaughtered animals here? I can't see a reference to it in the code?

Reply:
No, I was just referring to the element as it's presented on their web, namely "Producing Animals/Slaughtered", but here it doesn't make sense.